src/store/LastDigitStatsStore.ts

```diff
@@ -95,7 +95,7 @@ export default class LastDigitStatsStore {
     onMasterDataUpdate({ Close, tick }: TicksStreamResponse & { Close: number }) {
         if (!this.context || !this.mainStore.chart.currentActiveSymbol) return;
         this.lastTick = tick;
-        if (this.marketDisplayName !== this.lastSymbol) {
+        if (this.marketDisplayName !== this.lastSymbol && this.mainStore.lastDigitStats.isVisible) {
```

Comment:
Instead of this, can we add a condition at the top like the below line?

```
if (!this.mainStore.lastDigitStats.isVisible) return;
```

Reply:
It will prevent the whole logic in `onMasterDataUpdate` if the last digit stat is not shown.